src/Index.jsx

```diff
@@ -31,25 +33,34 @@ class Site extends React.Component {
   render() {
     const siteConfig = Object.assign({}, defaultSiteConfig, this.state.siteConfig);
     siteConfig.appConfig = Object.assign({}, defaultSiteConfig.appConfig, siteConfig.appConfig);
-    const { appConfig, navConfig, componentConfig } = siteConfig;
+    const { appConfig, componentConfig } = siteConfig;
 
     const { routeConfig, navigation } = routeConfiguration(siteConfig, componentConfig);
 
     const routes = Object.freeze(routeConfig);
+
```

Comment:
Since this is the entry point for the dev-site, to me it would make sense to move the creation of appLogo and nameConfig to the `App` component

Reply:
I'm trying to separate any static config outside of the `App` component. Long term I want to generate any static config prior to running webpack and creating the app. The app name and logo are static.